.gitmodules

```diff
@@ -2,3 +2,6 @@
 	path = libs/gutenberg-mobile
 	url = ../../wordpress-mobile/gutenberg-mobile.git
 	shallow = true
+[submodule "libs/portkey-android"]
+	path = libs/portkey-android
+	url = https://github.com/Automattic/portkey-android.git
```

Comment:
Thinking about this again, I think we want to use the relative url approach similar to how it's done for gutenberg-mobile because some users (specifically external contributors) might want to use the `https` version. Sorry for the double take on this.

Btw, when you are testing a url change, you'll want to run `git submodule sync` so it's picked up by `git`. I didn't know about that until today, so I thought it might be useful.

Reply:
Hello! Addressed in ea1263edaa52c8d17c53e75a1b41a6bc4a90b583